### DISCUSSION TITLE
MGnify vs UniRef alignment output

### DISCUSSION THREAD

**Discussion by pramblab** [facebookresearch/esm#424] [ESM Metagenomic Atlas]
Is there a way to fetch the mmseqs alignment between MGnify90 and UniRef? The data is part of the web interface and also discussed in detail in the manuscript (section "A.4.2."). 
It would be great, If the team shares the alignment output file or the scores as columns in metadata file.

**Comment by tomsercu** [comment]
We don't actually have the data for *all of MGnify90*, only for the 1M sample we used for analysis and atlas visualization, which is <1% of the full dataset. Even for this small fraction, the full alignment files with all uniref90 hits becomes a beast.
Fortunately mmseqs is blazing fast and really easy to use once you have sequences of interest.

**Reply by pramblab** [reply]
A quick follow up - Is it possible to share the list of 33,521 proteins that do not share similarity to UniRef90?
As discussed earlier, I have been running mmseqs2 alignment. This list would useful to compare my outputs with yours.

I referring to the  line in section "4. Evolutionary-scale structural characterization of metagenomics" -> "3.4% (33,521 proteins), no significant match is found in UniRef90 at all".

Thank you in advance!

**Comment by pramblab** [comment]
Thank you for the response. I could understand the limitations. 
I will try running it myself. Incase you have alignment for the MGnify30 dataset, please share it.
